src/PlotObject.tsx

```diff
@@ -4,24 +4,15 @@ import Plot from './Plot';
 import Axis from './components/Axis';
 import Legend from './components/Legend';
 import LineSeries from './components/LineSeries';
-import { AxisProps, LegendProps, LineSeriesProps, PlotProps } from './types';
+import { PlotObjectProps } from './types';
```

Comment:
not a big fan of this name, because it implies that it's the type of the props for the `PlotObject` component.